### PR TITLE
feat: added product listing page

### PR DESCRIPTION
**Pull request by NandhiniKrishnamoorthy**
Added product listing page
Please review and leave your comments

Preview link: https://sona-fancy-products.netlify.app/

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -5,17 +5,22 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"/>
     <!-- favicon -->
     <link rel="icon" href="https://img.icons8.com/flat-round/64/000000/crown--v1.png"/>
+    
     <!-- style 10 component library -->
     <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/padding.css" />
     <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/margin.css" />
     <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/typography.css" />
     <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/card.css" />
     <link rel="stylesheet"  href="https://styles-10-rating.netlify.app/css/ratings.css" />
```
Comment (pratik-kumar-sahu) [inline home.html:22]: - You can import `main.css` instead of individual imports. Please check if possible.

**Review by pratik-kumar-sahu — APPROVED** [review]
- Else looks good to be merged.
- UI looks nice.

**Review by Mahi24-prog — APPROVED** [review]
Looks good, Go ahead.

File: home.html (hunk 1 above)
Comment (NandhiniKrishnamoorthy) [reply home.html:22]: Since I have done some changes in main.css, I import all the required files

**Pull request merged**